app/products/calls/connection/connection.ts

```diff
@@ -201,8 +199,34 @@ export async function newConnection(
             }
         }
 
-        InCallManager.start({media: 'video'});
-        setSpeakerPhone(true);
+        InCallManager.start();
+        InCallManager.stopProximitySensor();
+
+        let btInitialized = false;
+        let speakerInitialized = false;
+
+        audioDeviceChanged = DeviceEventEmitter.addListener('onAudioDeviceChanged', (data: AudioDeviceInfoRaw) => {
+            const info: AudioDeviceInfo = {
```

Comment:
Does this also run upon joining calls?

Reply:
Yep, as far as I can tell, it fires at least once (when there's no bluetooth), and twice (second time when the bluetooth poll returns).